src/backend/core/api/serializers.py

```diff
@@ -148,3 +148,11 @@ def get_abilities(self, team) -> dict:
     def get_slug(self, instance):
         """Return slug from the team's name."""
         return instance.get_slug()
+
+
+class InvitationSerializer(serializers.ModelSerializer):
+    """Serialize invitations."""
+
+    class Meta:
+        model = models.Invitation
+        fields = ["email", "team", "role", "issuer"]
```

Comment:
issuer should be readonly and set programmatically from the logged-in user?

Reply:
I was unsure how and where to do that (serializer ? viewset ?). I finally opted for [this method](https://profil-software.com/blog/development/10-things-you-need-know-effectively-use-django-rest-framework/#using-currentuserdefault).

Reply:
I think we could handle this in the PR #50, where I create the route and the related tests.

Reply:
here ? https://github.com/numerique-gouv/people/pull/50/files#diff-78bd491985b92670ff5ca54200235d6db2810a6a1a999e673392a90a86e86c6fR159